hack/allocate.sh

```diff
@@ -179,8 +179,8 @@ registry() {
   # see https://kind.sigs.k8s.io/docs/user/local-registry/
 
   echo "${em}⑥ Registry${me}"
-  docker run -d --restart=always -p "127.0.0.1:50000:5000" --name "kind-registry" registry:2
-  docker network connect "kind" "kind-registry"
+  docker run -d --restart=always -p "127.0.0.1:50000:5000" --name "func-registry" registry:2
+  docker network connect "kind" "func-registry"
```

Comment:
The network created by kind is always named `kind` and does not collide (one per node regardless of the number of running clusters)